lib/player.js

```diff
@@ -322,6 +322,23 @@ goog.require('shaka.util.Timer');
  */
 
 
+/**
+ * @event shaka.Player.SessionDataEvent
+ * @description Fired when the manifest parsed find info about session data.
+ * @property {string} type
+ *   'sessiondata'
+ * @property {string} id
+ *   The id of the session data.
+ * @property {string} uri
+ *   The uri with the session data info.
+ * @property {string} language
+ *   The language of the session data.
+ * @property {string} value
+ *   The value of the session data.
```

Comment:
Since all of these are defined by HLS, it might be worth linking to the relevant part of the spec in the `@description` above.